packages/i18n/src/locales/ar_SA.json

```diff
@@ -132,8 +138,12 @@
     "playlist-exported": "{{name}} saved to local file",
     "queue": "Add playlist to queue",
     "rename": "Rename this playlist",
-    "songs": "{{total}} song",
-    "songs_plural": "{{total}} songs",
+    "songs_0": "{{total}} songs",
```

Comment:
This won't work since we use songs and songs_plural

Reply:
Done :) Just erased the `songs_X` lines and restored songs and `songs_plural` (in the 7 locales files where I found this). 

Not sure why that happened, though (I did not intend to modify locales files in this PR). 

PS: I did an extra commit for this because the first one was not passing the CI checks, some other files were missing the `songs` and/or `songs_plural` lines (but had no `songs_X` lines) so I added them there as well and then it passed.